docker/Dockerfile

```diff
@@ -0,0 +1,75 @@
+FROM centos:8
+# FROM python:3.8 
+# substitute centos:8 with oracle linux 8 (ask JPL Susan)
```

Comment:
Thank you for your feedback @rtburns-jpl . You said it's not necessary to decide on a base image now, but I guess it would be better to move now to a NASA-approved base image now, right? Dependencies requirements for this tool should be easy to meet for all base images. What would you suggest to use? Ubuntu 20.04?

Reply:
I agree with @gshiroma let's make a decision now and be done with it.

Reply:
Okay, if we want to make a decision now I'd say Ubuntu 20.04 is the best option.